integration-tests/infinispan/src/main/java/org/apache/camel/quarkus/component/infinispan/InfinispanResources.java

```diff
@@ -35,25 +44,50 @@
 
     @Inject
     RemoteCacheManager cacheManager;
+
     @Inject
     ProducerTemplate template;
 
+    @Inject
+    CamelContext camelContext;
+
     @PostConstruct
     public void setUp() {
         cacheManager.administration().getOrCreateCache(CACHE_NAME, (String) null);
     }
 
+    @Path("/inspect")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public JsonObject inspect() {
```

Comment:
I'm not sure about the intent. Would a mehod name like inspectCamelInfinispanClientConfiguration() better reflect the intent ?